xray_node/README.md

```diff
@@ -4,6 +4,6 @@ This subproject provides an interface to the `xray-core` library from JavaScript
 
 ## Building
 
-Because the target library looks up symbols from Node dynamically, it cannot be built with cargo directly without additional linker flags. See `scripts/build.js` for details.
+This project depends on the [`napi`](https://github.com/atom/napi) crate, which provides a safe interface to Node's N-API. It depends on the tandem Node.js package `napi` to provide a build harness. Running `npm build` will invoke that build hardness which sets up the environment variables and linker flags for Cargo.
```

Comment:
This URL points to a repo that doesn't exist, so I'll update it on master after merging.